app/main.py

```diff
@@ -15,5 +15,7 @@
     "Sergio": 3,
 }
 collection_of_coins = {1, 2, 25}
-
-# write your code here
+mutable = [my_favourite_films, marks, collection_of_coins]
```

Comment:
The `profile_info` variable is currently classified as mutable, but since it is a tuple, it should be classified as immutable. Please move `profile_info` to the `immutable` list.